pkgs/development/python-modules/vqgan-jax/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flax
+, jaxlib
+, pytorch
+, transformers
+}:
+
+buildPythonPackage rec {
+  pname = "vqgan-jax";
+  version = "unstable-2022-04-20";
+
+  src = fetchFromGitHub {
+    owner = "patil-suraj";
+    repo = "vqgan-jax";
+    rev = "1be20eee476e5d35c30e4ec3ed12222018af8ce4";
+    sha256 = "sha256-OZihAXpE0UsgauQ38XDmAF+lrIgz05uK0ro8SCdVsPc=";
+  };
+
+  propagatedBuildInputs = [
+    flax
+    jaxlib
+    transformers
+  ];
+
+  checkInputs = [
+    pytorch
```

Comment:
Either add `pytestCheckHook` or specify `checkPhase`.

Reply:
The setuptools check phase is already running by default

Reply:
We want to remove that functionality in the future.

Reply:
Okay, not sure what you want here. The package doesn't use pytest. How should I manually run the setuptools tests?

Reply:
You can use pytest anyway.

Reply:
If I use the pytestCheckHook, it runs 0 tests and fails.

Reply:
That's because there are no tests; `setuptoolsCheckPhase` doesn't run any either. Set `doCheck = false` (with comment) and add `pythonImportsCheck`.